eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantCoreInfo.java

```diff
@@ -0,0 +1,39 @@
+package uk.ac.ebi.eva.server.ws;
+
+import uk.ac.ebi.eva.commons.core.models.VariantType;
+
+import java.util.Map;
+import java.util.Set;
+
+public class VariantCoreInfo {
+    private String chromosome;
+
+    private long start;
+
+    private long end;
+
+    private String reference;
+
+    private String alternate;
+
+    private Set<String> ids;
+
+    private VariantType type;
+
+    private int length;
+
+    private Map<String, Set<String>> hgvs;
```

Comment:
Our HGVS algorithm is pretty primitive, so I would prefer to exclude that field from the response.